docker-compose.yml

```diff
@@ -1,74 +1,48 @@
-version: "3"
-
 services:
-  database-template:
+  database-flashcard:
     image: postgres:alpine
     restart: always
     network_mode: bridge
     expose:
-      - 1032
+      - 3032
     ports:
-      - "1032:5432"
+      - "3032:5432"
     volumes:
-      - templatedata:/var/lib/postgresql/data
+      - flashcarddata:/var/lib/postgresql/data
     environment:
       - POSTGRES_USER=root
       - POSTGRES_PASSWORD=root
-      - POSTGRES_DB=templatedatabase
+      - POSTGRES_DB=flashcard_service
```

Comment:
here you call it "flashcard_service" but in the application properties you call it "flashcardservice" This wont work, you need to name it the same